TestProject/UnitTest1.cs

```diff
@@ -8,11 +8,11 @@ public class UnitTest1
         public void Add_Test()
         {
             //Arrange 
```

Comment:
in the task, they said "Remember to refactor after each passing test.", you should keep the test cases for the previous steps but refactor them if needed.
also we can think about more non-straight forward test cases